package.json

```diff
@@ -51,10 +52,14 @@
     "test:e2e:chrome": "BROWSER=chrome yarn test:e2e",
     "test:e2e:firefox": "BROWSER=firefox yarn test:e2e",
     "test:e2e:firefox:ci": "HEADLESS=true yarn test:e2e:firefox",
-    "test:console": "jest ./test/console-smoke --config=test/jest-config/jest.config-console-smoke.js --maxWorkers=1 -t 'check console and browser wallet integrate'",
-    "deploy:console": "docker run -d --name console -e NX_VEGA_ENV=TESTNET -e NX_VEGA_URL=https://api.n08.testnet.vega.rocks/graphql -p 3000:80 vegaprotocol/trading:testnet",
+    "test:console": "jest ./test/console-smoke --config=test/jest-config/jest.config-console-smoke.js --maxWorkers=1",
+    "test:console:testnet": "ENV=testnet yarn test:console",
+    "test:console:mainnet": "ENV=mainnet yarn test:console",
+    "deploy:console:testnet": "docker run -d --name console -e NX_VEGA_ENV=TESTNET -e NX_VEGA_URL=https://api.n08.testnet.vega.rocks/graphql -p 3000:80 vegaprotocol/trading:testnet",
+    "deploy:console:mainnet": "docker run -d --name console -e NX_VEGA_URL=https://vega.aurora-edge.com/graphql -p 3001:80 vegaprotocol/trading:mainnet",
     "stop:console": "docker rm -f console",
-    "test:console:ci": "yarn deploy:console && yarn test:console && yarn stop:console",
+    "test:console:testnet:ci": "yarn deploy:console:testnet && yarn test:console:testnet && yarn stop:console",
+    "test:console:mainnet:ci": "yarn deploy:console:mainnet && yarn test:console:mainnet && yarn stop:console",
```

Comment:
I don't know how I feel about these being in the package.json. Console doesn't really have much impact in this project. I think perhaps two .sh files might be better for this one for test and one for mainnet.

4 long commands in order to run 2 tests seems a bit yuck. At the very least it should be easy enough to generalise these a lot.